hack/dev-setup-admission-azure.sh

```diff
@@ -18,22 +18,48 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Make it able to work on minikube and nodeless
+is_nodeless() {
+    nodes_len=$(kubectl get node -o json | jq ".items" | jq '. | length')
+    if [[ "$nodes_len" == "0" ]]; then
+      echo "true"
+      return
+    fi
+
+    echo "false"
+    return
```

Comment:
I guess the idiomatic bash way is:

```suggestion
    if [[ "$nodes_len" == "0" ]]; then
      return 0
    fi

    return 1
```

and

```bash
if is_nodeless; then

else

fi
```